contracts/btc-finality/src/finality.rs

```diff
@@ -458,11 +463,16 @@ pub fn index_block(
 pub fn tally_blocks(
     deps: &mut DepsMut,
     env: &Env,
-    activated_height: u64,
+    start_height: u64,
+    max_blocks_to_finalize: u64,
```

Comment:
```suggestion
```

can this function access the `MAX_FINALIZED_REWARDED_BLOCKS_PER_END_BLOCK` const directly?

Reply:
we can but this is to align with babylon's implementation. No strong opinion though